x-pack/plugins/observability/public/pages/alert_details/components/alert_summary.tsx

```diff
@@ -5,173 +5,151 @@
  * 2.0.
  */
 import React from 'react';
-import {
-  EuiText,
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiSpacer,
-  EuiPanel,
-  EuiTitle,
-  EuiLink,
-  EuiToolTip,
-  EuiIcon,
-  EuiButtonEmpty,
-} from '@elastic/eui';
+import { EuiText, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
-import { PageHeaderProps } from '../types';
+import moment from 'moment';
+import { ALERT_STATUS_ACTIVE, ALERT_STATUS_RECOVERED } from '@kbn/rule-data-utils';
+import { asDuration } from '../../../../common/utils/formatters';
+import { AlertSummaryProps } from '../types';
 import { useKibana } from '../../../utils/kibana_react';
 import { AlertStatusIndicator } from '../../../components/shared/alert_status_indicator';
+import { DEFAULT_DATE_FORMAT } from '../constants';
 
-export function AlertSummary({ alert }: PageHeaderProps) {
+export function AlertSummary({ alert }: AlertSummaryProps) {
   const { triggersActionsUi } = useKibana().services;
 
   return (
     <>
-      <EuiPanel color="subdued" hasBorder={false} paddingSize={'m'}>
-        <EuiTitle size="xs">
-          <div>
-            <FormattedMessage
-              id="xpack.observability.pages.alertDetails.alertSummary"
-              defaultMessage="Alert Summary"
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <EuiTitle size="xxs">
+            <h5>
+              <FormattedMessage
+                id="xpack.observability.pages.alertDetails.alertSummary.actualValue"
+                defaultMessage="Actual value"
+              />
+            </h5>
+          </EuiTitle>
+          <EuiSpacer size="s" />
+          <EuiText size="s" color="subdued">
+            {alert?.fields['kibana.alert.evaluation.value'] ?? '-'}
```

Comment:
Do you mean to use this file `packages/kbn-rule-data-utils/src/technical_field_names.ts` instead of hardcoding field names? If not, can you please elaborate on how to use `ParsedTechnicalFields`?

Reply:
I have removed hardcoded field names with fields from `@kbn/rule-data-utils`.

Reply:
Awesome 💯 👍🏻 !!